stable/artifactory-ha/requirements.yaml

```diff
@@ -1,5 +1,5 @@
 dependencies:
-- name: postgresql
-  version: 0.9.5
-  repository: https://kubernetes-charts.storage.googleapis.com/
-  condition: postgresql.enabled
+  - name: postgresql
+    version: 6.5.3
+    repository: https://kubernetes-charts.storage.googleapis.com/
+    condition: postgresql.enabled
```

Comment:
Is it possible to change this to .Values.postgresql.enabled ?
The reason being, this change would make it compatible to be embedded into other charts (like JCR).